basalt/nn/model.mojo

```diff
@@ -121,20 +98,18 @@ struct Model[
         # TODO: known copies (reference?)
         var outputs = List[Tensor[dtype]]()
         for i in range(len(g.outputs)):
-            outputs.append(self.parameters.params[g.outputs[i]])
+            outputs.append(TENSORS[g.outputs[i]])
         return outputs ^
 
     fn execute[num_nodes: Int](inout self, t_input: VariadicListMem[Tensor[dtype]]):
```

Comment:
Again, is there a way we can add an overload to take in a VariadicList of tensor collections instead of doing the logic here? Just thinking in the future if we have even more inputs.